.eslintignore

```diff
@@ -0,0 +1 @@
+/node_modules/*
```

Comment:
Is this necessary?

Reply:
Not sure, let me remove it and see what happens!

Reply:
Not needed, removing...